pkg/controller/glue/crawler/setup.go

```diff
@@ -71,24 +70,17 @@ func preObserve(_ context.Context, cr *svcapitypes.Crawler, obj *svcsdk.GetCrawl
 	return nil
 }
 
-func postObserve(_ context.Context, cr *svcapitypes.Crawler, obj *svcsdk.GetCrawlerOutput, obs managed.ExternalObservation, err error) (managed.ExternalObservation, error) {
+func postObserve(_ context.Context, cr *svcapitypes.Crawler, _ *svcsdk.GetCrawlerOutput, obs managed.ExternalObservation, err error) (managed.ExternalObservation, error) {
 	if err != nil {
 		return managed.ExternalObservation{}, err
 	}
 	cr.SetConditions(xpv1.Available())
 	return obs, nil
 }
 
-func postCreate(_ context.Context, cr *svcapitypes.Crawler, obj *svcsdk.CreateCrawlerOutput, _ managed.ExternalCreation, err error) (managed.ExternalCreation, error) {
```

Comment:
External name feature implementation wasn't correct so I fixed that as well - we can never use `cr.Name`, the source of truth is always the annotation.